src/caselawclient/models/documents/__init__.py

```diff
@@ -378,6 +378,13 @@ def validates_against_schema(self) -> bool:
         """
         return self.api_client.validate_document(self.uri)
 
+    def mint_fclid(self) -> None:
```

Comment:
Not sure this is a great name. "ensure_has_fclid"? (not sure I love that either)